plugins/woocommerce/client/admin/client/customize-store/style.scss

```diff
@@ -32,292 +35,116 @@
-		height: initial;
-	}
-}
-
-.cys-fullscreen-iframe {
-	display: block;
-	position: fixed;
-	top: 0;
-	left: 0;
-	width: 100%;
-	height: 100%;
-	border: none;
-	z-index: 9999;
-	transition: opacity 1.2s linear;
-	opacity: 0;
-}
-
-.woocommerce-customize-store__onboarding-welcome-modal,
-.woocommerce-customize-store__design-change-warning-modal,
-.woocommerce-customize-store__theme-switch-warning-modal {
-	width: 480px;
-	max-width: 480px;
-
-	.components-modal__header {
-		padding-top: 32px;
-	}
-
-	p {
-		padding: 16px 0 16px 0;
-		margin: 0;
-	}
-
-	a,
-	button {
-		text-decoration: none !important;
-	}
-
-	.components-button {
-		padding: 8px 16px;
-	}
 
-	h1 {
-		line-height: 28px;
-		font-size: 20px;
-		color: #1e1e1e;
-	}
-
-	span {
-		color: #1e1e1e;
-		font-weight: 700;
-	}
-
-	.woocommerce-customize-store__design-change-warning-modal-footer,
-	.woocommerce-customize-store__theme-switch-warning-modal-footer {
-		display: flex;
-		gap: 12px;
-		justify-content: flex-end;
-	}
-}
-
-.woocommerce-customize-store__opt-in-usage-tracking-modal {
-	width: 480px;
-	max-width: 480px;
-
-	a {
-		text-decoration: underline;
-		color: inherit;
-	}
-
-	.components-modal__header {
-		padding: 32px 32px 16px 32px;
-	}
-
-	.components-modal__content {
-		padding: 0;
-	}
-
-	.core-profiler__checkbox {
-		padding: 16px 32px 16px 32px;
-
-		div {
-			display: flex;
+		.woocommerce-customize-store-banner-content {
+			width: 100%;
+			max-width: 60%;
 
-			.components-checkbox-control__input-container {
-				display: block;
+			@media screen and (max-width: 1024px) {
+				max-width: 100%;
 			}
-		}
-	}
-
-	.woocommerce-customize-store__design-change-warning-modal-footer {
-		border-top: 1px solid #ddd;
-		padding: 24px 32px 32px 32px;
-		width: 100%;
-		display: flex;
-		gap: 12px;
-		justify-content: flex-end;
 
-		button.is-link {
-			padding: 6px 12px;
-			text-decoration: none;
-		}
-	}
-
-}
-
-.woocommerce-onboarding-loader {
-	min-height: 100vh;
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	flex-direction: column;
-	background-color: #fff;
-
-	.woocommerce-onboarding-loader-wrapper {
-		align-items: flex-start;
-		display: flex;
-		flex-direction: column;
-		justify-content: center;
-		max-width: 520px;
-		min-height: 400px;
+			button.components-button {
+				padding: 8px 16px;
+				height: 40px;
+			}
 
-		.woocommerce-onboarding-loader-container {
-			text-align: center;
-			min-height: 400px;
-		}
+			h1,
+			h2 {
+				font-size: 1.25rem;
+				line-height: 23.87px;
+				font-weight: 500;
+				color: #000;
+				padding: 0;
+			}
 
-		.loader-hearticon {
-			position: relative;
-			top: 2px;
-			left: 2px;
-		}
-	}
-}
+			p {
+				margin: 12px 0 28px 0;
+				color: #2f2f2f;
+				line-height: 16px;
+			}
 
-.woocommerce-customize-store__step-designWithoutAi {
-	.woocommerce-onboarding-loader-wrapper {
-		min-height: auto;
-		.woocommerce-onboarding-loader-container {
-			min-height: auto;
+			.pick-your-theme-banner__content {
```

Comment:
I believe this CSS selector is only [used in `intro-banners.tsx`](https://github.com/woocommerce/woocommerce/blob/2ef538ce2cfc6a9b1cc9f62512ec2dc4f97fb195/plugins/woocommerce/client/admin/client/customize-store/intro/intro-banners.tsx#L260), which has been removed, no? Could we remove these styles too?

Reply:
Oops, you're right. GitHub had collapsed that file by default, and I missed it. :man_facepalming: